pkgs/desktops/plasma-5/plasma-desktop/default.nix

```diff
@@ -58,7 +58,14 @@
 , runCommandLocal
 , makeWrapper
 }:
-
+let
+  # run gsettings with desktop schemas for using in kcm_accces kcm
```

Comment:
Typo here?

Reply:
if you are meaning `kcm_accces kcm` part it is because `kcm_access` is the name of `kcm`

added `"` around `kcm_access` to make it apparent